tailwind.config.js

```diff
@@ -1,7 +1,4 @@
 module.exports = {
-  content: ["./index.html", "./src/**/*.fs"],
-  theme: {
-    extend: {},
-  },
-  plugins: [require("daisyui")],
-};
\ No newline at end of file
+  content: ['./index.html', './js/**/*.js'],
```

Comment:
Typical pattern with non js frontends is to still parse tailwind out of the source files not out of the generated JS so that the tailwind build is triggered on file edit - not on build completion